### ISSUE TITLE
Add ebgp-multihop for BGP to DO

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-declarative-onboarding#236]
### Is your feature request related to a problem? Please describe.
Not all BGP api is available through DO. I need the ability to setup bgp peering across networks and need ebgp-multihop.

### Describe the solution you'd like
Example BGP output below using "ebgp-multihop 2"

```
router bgp 65530
 bgp graceful-restart restart-time 120
 aggregate-address 10.0.0.0/16 summary-only
 aggregate-address 10.1.0.0/16 summary-only
 aggregate-address 10.2.0.0/16 summary-only
 aggregate-address 10.3.0.0/16 summary-only
 redistribute kernel
 neighbor 10.255.255.4 remote-as 65515
 neighbor 10.255.255.4 ebgp-multihop 2
 neighbor 10.255.255.5 remote-as 65515
 neighbor 10.255.255.5 ebgp-multihop 2
!
```

### Describe alternatives you've considered
Have to run tmsh commands in runtime init like this...

```
tmsh create net routing bgp 65530 local-as 65530 neighbor add { 10.255.255.4 { remote-as 65515 ebgp-multihop 2 } 10.255.255.5 { remote-as 65515 ebgp-multihop 2 } } address-family { ipv4 { redistribute add { kernel } aggregate-address add { 10.0.0.0/16 { summary-only enabled } 10.1.0.0/16 { summary-only enabled } 10.2.0.0/16 { summary-only enabled } 10.3.0.0/16 { summary-only enabled } } } } graceful-restart { restart-time 120 }
```

### Additional context
Add any other context, such as the desired tmsh configuration, about the feature request here.

**Comment by dstokesf5** [comment]
Thank you for your feedback. Your feature request has been added to the internal product backlog as AUTOTOOL-2668.

**Comment by dstokesf5** [comment]
This was resolved with the 1.24.0 release of Declarative Onboarding.
